docs/topics/router/index.rst

```diff
@@ -4,7 +4,7 @@ RapidSMS Routers
 
 .. module:: rapidsms.router
 .. module:: rapidsms.router.base
-.. module:: rapidsms.router.blocking
```

Comment:
Just curious, why the change to this line and adding `noindex`?

Reply:
Yeah good question, flake8 was complaining about it being a duplicate.